Docs 10708 3.9 devkit important (#36)
mulesoft/docs-connector-devkit: modules/ROOT/pages/connector-connection-strategies.adoc

endif::[]
:keywords: devkit, strategy, connector, oauth, config, connection, authentication

include::partial$devkit-important.adoc[]

> Since DevKit 3.6, connections are no longer supported at the `@Connector` level (excluding OAuth V1), but now are defined in a "configuration" class and injected into the `@Connector`.

Reviewer: ```suggestion
Connections are no longer supported at the `@Connector` level (excluding OAuth V1) starting with DevKit 3.6. Connections are now defined in a "configuration" class and injected into the `@Connector`.
```
